zerver/lib/actions.py

```diff
@@ -2302,11 +2303,11 @@ def do_change_default_language(user_profile, setting_value, log=True):
         log_event(event)
     send_event(event, [user_profile.id])
 
-def set_default_streams(realm, stream_names):
-    # type: (Realm, Iterable[text_type]) -> None
+def set_default_streams(realm, stream_names, stream_descriptions):
```

Comment:
This will probably be less fragile if this is a stream dict like below (or at least a list of tuples). Then length differences don't cause silent bugs.